src/lazy/expanded/macro_table.rs

```diff
@@ -5,13 +5,23 @@ use std::collections::HashMap;
 /// This list parallels
 /// [`MacroExpansionKind`](crate::lazy::expanded::macro_evaluator::MacroExpansionKind),
 /// but its variants do not hold any associated state.
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
 pub enum MacroKind {
     Void,
     Values,
     MakeString,
 }
 
+impl MacroKind {
+    fn name(&self) -> &str {
+        match self {
+            MacroKind::Void => "void",
+            MacroKind::Values => "values",
+            MacroKind::MakeString => "make_string",
+        }
+    }
+}
```

Comment:
🗺️ Per feedback, I added this method and used it to initialize the `MacroTable`.